app/models/miq_queue.rb

```diff
@@ -501,23 +501,6 @@ def unfinished?
     !finished?
   end
 
-  def self.atStartup
```

Comment:
What's the reason this class doesn't need `include_concern 'AtStartup'` too? I think I'm missing something.